README.md

```diff
@@ -7,16 +7,16 @@ Portfolio: setup a mobile version
 ## Built With
 
 - HTML5
-- CSS
+- CSS (FlexBox and Grid combined)
 - GIT / GITHUB
 
 ## Authors
 
 👤 **Wolf M. Jacques**
 
-- GitHub: Wolfmikerson
-- Twitter: @wolfmikerson
-- LinkedIn: [LinkedIn] https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
+- GitHub: Wolfmikerson (https://github.com/Wolfmikerson)
+- Twitter: @wolfmikerson (https://twitter.com/MikersonWolf)
+- LinkedIn: [Wolf_LinkedIn](https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
 
```

Comment:
- [ ]  [OPTIONAL] For Github and Twitter, kindly do the same structure as your LinkedIn profile.